bindings_node/src/client.rs

```diff
@@ -211,14 +214,21 @@ impl Client {
   }
 
   #[napi]
-  pub async fn can_message(&self, account_addresses: Vec<String>) -> Result<HashMap<String, bool>> {
-    let results: HashMap<String, bool> = self
+  /// The resulting vec will be the same length as the input and should be zipped for the results.
+  pub async fn can_message(&self, account_identities: Vec<PublicIdentifier>) -> Result<Vec<bool>> {
```

Comment:
can we return a `HashMap` again so it's easy to associate the state with the identifier?

Reply:
perhaps the key could be `${identifier_kind}/${identifier}`?

Reply:
Only did the identifier. The key should be unique enough.